ee/api/conversation.py

```diff
@@ -5,7 +5,6 @@
 from django.http import StreamingHttpResponse
```

Comment:
@skoob13 Are visual regression tests just fail randomly from time to time? The code looks alright, so I'm not sure what's causing it.